README.md

```diff
@@ -426,6 +426,11 @@ Platform-specific variants also available:
     CIBW_TEST_REQUIRES: nose
     CIBW_TEST_COMMAND: "nosetests {project}\\tests"
 </code></pre></td>
+<td><i>example using pytest requirements.txt</i><pre><code>environment:
```

Comment:
Move to the "Environment variable: `CIBW_TEST_REQUIRES`" section a few lines above, rather than adding it here as an example of "YML syntax"?